exercises/bank-account/HINTS.md

```diff
@@ -0,0 +1,4 @@
+## Hints
+
+This exercise is testing mutable state that can be accessed saftely from multiple threads. Scala provides a variety of ways to protect 
+mutable state. For developers familiar with Java concurrency, Scala can utilize the Java concurrency support such as the Java synchronized block.
```

Comment:
The suggested addendum by @abo64 seems like a good one, even though it is a bit of a spoiler.